### DISCUSSION TITLE
Using nuxtServerinit in Nuxtjs 3

### DISCUSSION THREAD

**Discussion by amirtbi** [nuxt/nuxt#17114] [Questions]
Hi.
I'm using the Pinia in My Nuxtjs3 app. It works properly, but I'd like to know that every time that the method Create is called using Pinia, the rendered data is loaded through the server, not just from client. Is it possible to check that?
And Does new version of Nuxtjs support nuxtServerInit?
nuxt/framework#5304

**Comment by valqelyan** [comment]
any updates ?

**Reply by kissu** [reply]
I think that you may get some solutions on Pinia's github or by using `VueUse`. 👍🏻
